opal/util/arch.h

```diff
@@ -195,33 +195,41 @@
 #define OPAL_ARCH_UNUSEDMASK  0xfc000000 /* mark the unused fields */
 
 /* BYTE 1 */
-#define OPAL_ARCH_ISBIGENDIAN 0x00000008
+#define OPAL_ARCH_ISBIGENDIAN 0x00000008  /* b 0000 0100 */
```

Comment:
What is the interest of the added text, it does not read as a "clarification" to me ?

Reply:
Also, why was there a need to change the values for these defines ? And add an explicit one for the missing case (such as OPAL_ARCH_LONGIS32) ?

Reply:
I agree the extra comment isn't needed on the ISBIGENDIAN, and the changes to logical/bool were unnecessary.

The previous code was
```
#define OPAL_ARCH_LONGISxx     0x0000c000 /* mask for sizeof long */
#define OPAL_ARCH_LONGIS64     0x00001000
```
so the ISxx mask `c000` doesn't even contain its member `1000`, so that part needs changed to `4000`, so I'll reduce the change to just that part.